runtime/doc/news.txt

```diff
@@ -15,6 +15,8 @@ BREAKING CHANGES                                                *news-breaking*
 
 The following changes may require adaptations in user config or plugins.
 
+• colon (:) has been removed from |'isfname'| on Windows.
```

Comment:
The news item should give a hint about the meaning of the change, e.g. "File path drive letters are now detected on Windows without needing to explicitly put ':' in 'isfname'." 

Also need to update `:h nvim-defaults`.